src/side-navigation/nav-item.js

```diff
@@ -48,7 +48,7 @@ class NavItem extends React.Component<NavItemPropsT> {
     };
     return (
       <NavLink
-        href={item.itemId}
+        href={item.disabled ? null : item.itemId}
```

Comment:
I think it shouldn't be a link element if disabled.

Reply:
Thank you. Updated to span if the item is disabled